kahuna/public/js/components/gr-top-bar/gr-top-bar.css

```diff
@@ -2,7 +2,7 @@ gr-top-bar {
     display: block;
     height: 50px; /* faking we're in the flow */
     position: relative;
-    z-index: 3;
+    z-index: 30;
```

Comment:
Absolutely, sounds like the best approach. But in absence of a layer reference, we will need to just check manually that id doesn't interfere with other overlays (dnd, auth error, advanced search, etc)